File: Problem Set 1/mario-less.c
#include <stdio.h>
#include <cs50.h>

void print_row(int spaces, int bricks);

int main(void)
{
    // Ask the user for pyramid height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);

    // Print pyramid height
    for (int i = 0; i < n; i++)
    {
        // Print rows of bricks
        print_row (n - i - 1, i + 1);
    }
}

void print_row(int spaces, int bricks)
{
    // Spaces
    for (int i = 0; i < spaces; i++)
    {
        printf(" ");
    }
    // Bricks
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: Problem Set 6/mario-less.py
from cs50 import get_int

def main():
    height = get_height()

    for i in range (1, height + 1):
        print(" " * (height - i) + "#" * i)

def get_height():
    while True:
        height = get_int("Height: ")
        if 1 <= height <= 8:
            return height

if __name__ == "__main__":
    main()
